packages/lab/src/static-list/StaticListItem.css

```diff
@@ -0,0 +1,13 @@
+.saltStaticListItem {
+  list-style-type: none;
+  color: var(--salt-content-primary-foreground);
+  display: flex;
+  gap: var(--salt-spacing-100);
+  box-sizing: border-box;
+  padding: var(--salt-spacing-50) var(--salt-spacing-100);
+  min-height: calc(var(--salt-size-base) + var(--salt-spacing-100));
+}
+
+.saltStaticListItem .saltIcon {
+  min-height: var(--salt-size-base);
```

Comment:
icon would have base size? is this a workaround for some other problem?

Reply:
it is a workaround, will check with @ivan-calderon about the nuances of the css implementation, a padding might be more intuitive, else i can add some context in a comment

Reply:
How does this affect icon buttons?

Reply:
_This will affect icon buttons and any internal button in the content, we should probably target immediate children of the main component only to avoid suprises_. will update.